app/assets/stylesheets/events.scss

```diff
@@ -308,6 +318,24 @@ label {
       list-style: square;
       font-size: 8pt;
       text-align: left;
+      color: $main-color-5;
+    }
+
+    td.buttons {
+      text-align: center;
+    }
+
+    input {
+      background: none;
+      color: $main-color-5;
```

Comment:
Properties should be ordered background, border, border-color, border-radius, color